src/styles/blocks/catalog.scss

```
@@ -0,0 +1,35 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, minmax($card-size, 1fr));
+  justify-content: center;
+  justify-items: center;
+  gap: $gap;
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, minmax($card-size, 1fr));
+    }
```

```suggestion
    grid-template-columns: repeat(2, $card-size);
```

Too many spaces between cards

Fix everywhere